app.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express");
+const app = express();
+const endpointsJson = require("./endpoints.json");
+
+app.use(express.json());
```

Comment:
Please only submit code relevant to the ticket you're working on. Unless you're working on a post or patch request, `app.use(express.json())` is outside of the scope of this ticket.